src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { InputFieldComponent } from '@styleguide';
+import { MatButtonModule } from '@angular/material/button';
+import { FormsModule } from '@angular/forms';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'sg-filter-dropdown',
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatIconModule,
+    MatMenuModule,
+    MatCheckboxModule,
+    InputFieldComponent,
+    MatButtonModule,
+    FormsModule,
+  ],
+  templateUrl: './filter-dropdown.component.html',
+  styleUrls: ['./filter-dropdown.component.scss'],
+})
+export class FilterDropdownComponent implements OnInit {
+  selectedItems: string[] = [];
+  @Input() leadingIcon: string = '';
+  @Input() hasSearch: boolean = true;
+  @Input() disabled: boolean = false;
+  @Input() menuLabel!: string;
+  @Input() menuItems!: string[];
+  @Input() activeFilter = false;
+  @Output() updateSelection = new EventEmitter();
```

Comment:
would be nice to add some comments to this (now that storybook makes it pretty to read!)